db/db_logical_block_size_cache_test.cc

```diff
@@ -90,6 +90,7 @@ TEST_F(DBLogicalBlockSizeCacheTest, OpenClose) {
     ASSERT_EQ(0, cache_->Size());
     delete db;
   }
+  ASSERT_OK(DestroyDB(dbname_, options, {}));
```

Comment:
How about in the destructor?

Reply:
different tests have different setups of dbname and column families. So cannot just call DestroyDB with the same setup in destructor.